src/ui/FormElement.tsx

```diff
@@ -56,7 +58,7 @@ const FormElement = ({
     >
       {label && (
         <Label
-          variant={LabelVariants.BOLD}
+          variant={labelIsBold ? LabelVariants.BOLD : LabelVariants.NORMAL}
```

Comment:
I would just pass `labelVariant` as a prop instead with a defaultProp set to `LabelVariants.BOLD`

Reply:
Good suggestion 👍